### ISSUE TITLE
beta_test_v1.2.0

### ISSUE THREAD

**Issue by emoose** [nipkownix/re4_tweaks#406]
...wat? https://steamdb.info/app/254700/depots/
![image](https://user-images.githubusercontent.com/1755499/209665675-743dfa42-cb7e-4e57-9c8a-24608e6c9ead.png)
Anyone heard something about this?

**Comment by nipkownix** [comment]
Hopefully they're not adding a promotion screen to make people buy RE4 Remake (like they did in RE2 Remake when RE3 Remake came out...) .-.

**Comment by linkthehylian** [comment]
Holy shit?! I wasn't even aware a new beta branch was recently added until clicking this issue.

I wonder what will get changed, assuming something even **does** get changed lmao

**Comment by emoose** [comment]
IIRC there were some font changes in the newer console ports right? Maybe they'll finally bring those to PC.

What would be really sweet is if they ported some of the Xbone version stuff over, think I've heard Xbone is DX11/DX12 only (and x64 only too maybe?), so they probably have all the code ready to support those, maybe new renderer would be able to fix some of the broken effects here.

Being so close to RE4make release I'm not really expecting any big improvements though, most likely just adding some kind of promo like nipkow mentioned...

**Comment by linkthehylian** [comment]
The font changes would be a really nice addition because I actually prefer how it looks on console compared to PC. I'm not getting my hopes up, but it would be cool if a lot of things were added/improved.

> think I've heard Xbone is DX11/DX12 only (and x64 only too maybe?)

I wasn't aware of that either. That basically means that the PC version has been outdated for years. Neat, lol

**Comment by nr1995** [comment]
@nipkownix  What does v1.1.0 do?

**Comment by emoose** [comment]
@nr1995 just added Chinese language support and some really minor bugfixes, they messed up the release on Steam though so some owners get 1.1.0 and others 1.0.6, hopefully this update will actually be released properly this time...

**Comment by linkthehylian** [comment]
IIRC he mentioned something a while back in another issue about buying RE4 during a certain period. New owners basically get **1.1.0**, and older owners are stuck with **1.0.6** (this is also the version the JP executable is on).

Feel absolutely free to correct me if I'm misremembering lol

**Comment by nr1995** [comment]
I bought very early and am stuck on v1.0.6

**Comment by RiasatSalminSami** [comment]
Hopefully they will add HDR support. Afaik the ps4 version running on ps5 has fake HDR. Correct me if I'm wrong.

**Comment by nipkownix** [comment]
Oh, hmm, I haven't noticed any updates, but I'll verify the game's cache just to make sure.

**Comment by linkthehylian** [comment]
Nothing on my end either. Hmm.

**Comment by emoose** [comment]
>Maybe there was never any real update in the first place ._.

Yeah seems not, posted what I could find about it at https://github.com/nipkownix/re4_tweaks/issues/473#issuecomment-1447731323, looks like it's just a change to the signature on the EXE for some reason (even though the signature expiry date was years ago...), maybe they had a signing key leak or something.

At least they finally fixed the 1.1.0 issue, just too bad whatever way they did it broke a lot of RE4HD installs though >.>

**Comment by ShadowOne333** [comment]
I'vbeegetting a reproduceable crash with the latest update. on Professional, trying to get back after killing Bitorez crashes the game each time. Here's the crash log for it:
[bio4.exe.20230304122217.zip](https://github.com/nipkownix/re4_tweaks/files/10889473/bio4.exe.20230304122217.zip)

And the prompt:
![20230304122222_1](https://user-images.githubusercontent.com/13888698/222925272-faf92d5c-8c92-4388-a867-812de550ebb6.jpg)
